test/auto/injectorSpec.js

```diff
@@ -237,6 +237,15 @@ describe('injector', function() {
     });
 
 
+    // Only Chrome and Firefox support this syntax.
+    if (/chrome|firefox/i.test(navigator.userAgent)) {
+      it('should be possible to annotate functions that are declared using ES6 syntax', function() {
+        // The function is generated using `eval` as just having the ES6 syntax can break some browsers.  
+        expect(annotate(eval('({ fn(x) { return; } })').fn)).toEqual(['x']);
```

Comment:
Alright, I'd rather we jUst put the test in a different sOurce file which is only loaded in known supported browsers, because I really don't want to rely on eval. Take your pick and document clearly the approach you've taken. Otherwise LGTM